corehq/apps/geospatial/filters.py

```diff
@@ -0,0 +1,22 @@
+from django.utils.translation import gettext as _
+from django.utils.translation import gettext_lazy
+
+from corehq.apps.reports.filters.base import BaseSingleOptionFilter
+
+
+class GPSDataFilter(BaseSingleOptionFilter):
```

Comment:
Should we have a `help_text` field here to describe what this filter is doing? I'm wondering if users might get confused as to what filtering for "All GPS Data" means.

Reply:
Cool, will do that.

Reply:
https://github.com/dimagi/commcare-hq/pull/33538/commits/fa70fbf049ab917cd4437df8581217f098f08003